consensus/dummy/consensus.go

```diff
@@ -334,6 +337,18 @@ func (eng *DummyEngine) FinalizeAndAssemble(chain consensus.ChainHeaderReader, h
 	}
 	header.Extra = append(extraPrefix, header.Extra...)
 
+	// Set the min delay excess
+	minDelayExcess, err := customheader.MinDelayExcess(
```

Comment:
For my own understanding, is there a reason why we don't use `customheader.ExtraPrefix` for handling setting the min delay excess as we already do for the desired gas target excess?